coverage/lcov-report/src/middlewares/authenticate.ts.html

```
@@ -178,7 +178,7 @@ <h1><a href="../../index.html">All files</a> / <a href="index.html">src/middlewa
             <div class='footer quiet pad2 space-top1 center small'>
```

HTMLHint (1.5.0)

[error] The value of attribute [ class ] must be in double quotes.

(attr-value-double-quotes)